srcpkgs/libdivecomputer-subsurface/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'libdivecomputer-subsurface'
+pkgname=libdivecomputer-subsurface
+version=5.0.5
+revision=1
+wrksrc="libdivecomputer-subsurface-branch-${version}"
+build_style=gnu-configure
+hostmakedepends="libtool automake autoconf pkg-config"
+short_desc="Subsurface version of libdivecomputer"
+maintainer="Alan Brown <adbrown@rocketmail.com>"
+license="LGPL-2.1-only"
+homepage="https://subsurface-divelog.org/"
+distfiles="${homepage}/downloads/${pkgname}-branch-${version}.tgz"
+checksum=94b69d1b12d3c35376fdce96088c0cce21392a37a0ff67991246171d22055c02
+shlib_provides="libdivecomputer.so"
```

Comment:
Why `shlib_provides`?

Reply:
Because it is a fork of libdivecomputer even though this is not on void. I must admit I wasn't sure whether to add or not.

Reply:
Removed this